main.go

```diff
@@ -58,23 +58,35 @@ func addAdmin(app *gin.Engine, conf *config.Type) {
 	// for testing users, proxy admin to target because the build output has not been embed
 	dist := getSub(admin, "admin", "dist")
 	index, err := dist.Open("index.html")
+
+	var router gin.IRoutes = app
+	if len(conf.AdminBasePath) > 0 {
```

Comment:
If conf.AdminBasePath default to '', then router is nil and may cause problems in the following

Reply:
router is initialized as app. 
```
var router gin.IRoutes = app
```